api/controller.py

```diff
@@ -650,13 +652,27 @@ def load_licensepools(self, library, identifier_type, identifier):
         """
         _db = Session.object_session(library)
         pools = (
-            _db.query(LicensePool)
-            .join(LicensePool.collection)
-            .join(LicensePool.identifier)
-            .join(Collection.libraries)
-            .filter(Identifier.type == identifier_type)
-            .filter(Identifier.identifier == identifier)
-            .filter(Library.id == library.id)
+            _db.scalars(
+                select(LicensePool)
```

Comment:
Took the opportunity where I could to replace `query` with sqlalchemy 2.x style `select`, to make that move easier when we get to it.